WordPress/src/test/java/org/wordpress/android/ui/reader/viewmodels/ReaderPostDetailViewModelTest.kt

```diff
@@ -321,12 +362,17 @@ class ReaderPostDetailViewModelTest {
     @Test
     fun `given global related posts fetch succeeds, when related posts requested, then global related posts shown`() =
             test {
-                val globalRelatedPostsUiState = createDummyRelatedPostsUiState(isGlobal = true)
+                val globalRelatedPostsUiState = createDummyRelatedPostsUiState(
+                        relatedPosts = globalRelatedPosts,
+                        isGlobal = false
```

Comment:
Minor (🔍): The `isGlobal` here might need to be changed to `true` just for completeness purposes. As a suggestion, I usually extract the `given` in its own parameter when that makes sense to give the reader an immediate feedback that this is what is being tested. As such and with further fine tuning this test and the below can benefit from that and the merging of the `local/globalRelatedPosts` fields I mentioned in another comment of mine. In the end it can look something like:
```
    @Test
    fun `given local related posts fetch succeeds, when related posts are requested, then local related posts shown`() =
            test {
                val localRelatedPostsUiState = relatedPostsUiState(isGlobal = false)
                val uiStates = init().uiStates

                viewModel.onRelatedPostsRequested(readerPost)

                assertThat(uiStates.last().localRelatedPosts).isEqualTo(localRelatedPostsUiState)
            }

    @Test
    fun `given global related posts fetch succeeds, when related posts requested, then global related posts shown`() =
            test {
                val globalRelatedPostsUiState = relatedPostsUiState(isGlobal = true)
                val uiStates = init().uiStates

                viewModel.onRelatedPostsRequested(readerPost)

                assertThat(uiStates.last().globalRelatedPosts).isEqualTo(globalRelatedPostsUiState)
            }
```